mlrun/db/httpdb.py

```diff
@@ -275,6 +276,10 @@ def connect(self, secrets=None):
             config.artifact_path = config.artifact_path or server_cfg.get(
                 "artifact_path"
             )
+            config.feature_store.data_prefixes.default = (
```

Comment:
Since we changed to the full dict - you cannot just place it in config.feature_store.data_prefixes.default anymore.